base.py

```diff
@@ -420,8 +428,12 @@ def denormalize_name(self, name):
         return name
 
     def _denormalize_quote_join(self, *idents):
+        split_idents = reduce(
+            operator.add,
+            [ids.split('.') for ids in idents if ids is not None])
+        split_idents = [ids.strip('"') for ids in split_idents]
```

Comment:
A element of the list of identifiers may be already reconstructed, so split them again here.